tests/storage/assert_storage.rs

```diff
@@ -0,0 +1,257 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::sync_storage::SyncStorage;
+use kvproto::kvrpcpb::{Context, LockInfo};
+use tikv::storage::{self, Key, KvPair, Mutation, make_key};
+use tikv::storage::mvcc::{self, MAX_TXN_WRITE_SIZE};
+use tikv::storage::txn;
+use raftstore::cluster::Cluster;
+use raftstore::server::ServerCluster;
+use tikv::util::HandyRwLock;
+use super::util::new_raft_storage_with_store_count;
+
+#[derive(Clone)]
+pub struct AssertionStorage {
+    pub store: SyncStorage,
+    pub ctx: Context,
+}
+
+impl Default for AssertionStorage {
+    fn default() -> AssertionStorage {
+        AssertionStorage {
+            ctx: Context::new(),
+            store: SyncStorage::new(&Default::default()),
```

Comment:
Since I  moved this block from test_storage and it used Default::default. However I prefer to consider your tip and  use Config::default() to make it more clear .